test/tests/reporter/GriddedDataReporter/tests

```diff
@@ -0,0 +1,8 @@
+[Tests]
+  [gReporterexo]
+    type = JSONDiff
+    input = 'griddedData.i'
+    jsondiff= 'griddedData_out.json'
+    detail = 'diffing the data stored in the GriddedDataReporter'
```

Comment:
Don't use detail, use requirement.  Detail is used to append a detail onto a requirement in a test.  I'm not sure what it will do if its by itself.  Add some more information to the requirement like this:
requirement = "This tests that a reporter is able to read in a gridded data file and produce the appropriate grid from it that can be used in PiecewiseMultiInterpolation like functions."

Reply:
done